code/src/java/plugin/lsttokens/NaturalattacksLst.java

```diff
@@ -122,186 +122,160 @@ protected ParseResult parseTokenWithSeparator(LoadContext context,
 			{
 				return pr;
 			}
-			Equipment anEquip = createNaturalWeapon(context, obj, tokString.intern());
+			String wpn = tokString.intern();
 
-			if (anEquip == null)
+			StringTokenizer commaTok = new StringTokenizer(wpn, Constants.COMMA);
+
+			int numTokens = commaTok.countTokens();
+			if (numTokens < 4)
 			{
-				return ParseResult.INTERNAL_ERROR;
-				//return new ParseResult.Fail("Natural Weapon Creation Failed for : "
-				//		+ tokString, context);
+				return new ParseResult.Fail("Invalid Build of " + "Natural Weapon in "
+						+ getTokenName() + ": " + wpn, context);
 			}
 
-			if (count == 1)
+			String attackName = commaTok.nextToken();
+
+			if (attackName.equalsIgnoreCase(Constants.LST_NONE))
 			{
-				anEquip.setModifiedName("Natural/Primary");
+				return new ParseResult.Fail("Attempt to Build 'None' as a "
+						+ "Natural Weapon in " + getTokenName() + ": " + wpn, context);
 			}
-			else
+
+			attackName = attackName.intern();
+			Equipment anEquip = new Equipment();
```

Comment:
Is this really `anItem` ?

Reply:
It's specifically Equipment - it HAS to be since the getEquipmentHead is called on it later.  So I think a more generic names is a disservice to the reader.  Now whether it should be "anEquipment" or "equipment" or "naturalWeapon" (the last probably being most appropriate) is up for discussion

Reply:
Aha!  `naturalWeapon` would be perfect.